randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -1345,17 +1381,16 @@ Extra - asset_id: 114
   * Layers: default
   * Normal Door to Warehouse Zeela Room/Door to Warehouse Energy Tank Room
   > Pickup (Energy Tank)
-      Any of the following:
-          Missile or Super Missile
-          All of the following:
-              Morph Ball
-              Morph Ball Bombs or Power Bombs
+      Trivial
 
 > Pickup (Energy Tank); Heals? False
   * Layers: default
   * Pickup 43; Category? Major
   > Door to Warehouse Zeela Room
```

Comment:
i would put all this logic to *get* the etank, not leave from it

Reply:
The door locks behind you, I think. So having the requirements made sense to me this way because you have to meet them to get out, not to get in.

Reply:
the issue i see with it, is that it can become logical then for you to get that item hitless, and then get an etank there. and pretty sure the frog-thingies damage you early enough that that would be impossible

Reply:
I moved it. The item could also be the thing that lets you defeat the Beetoms.

Reply:
oh right. did not think of that hmm.
i think the most future proof thing would be to have two different requirements then, one very low one that assumes you dont need to kill the beetoms, and one that assumes you *have* to kill them.

But we could also just //TODO: that.

Reply:
Split the requirements. Just need to survive long enough to get the item one way, and then defeat the Beetoms to get out. Worst case scenario trickless has Normal Damage >= 100 + a way to defeat them.

![image](https://github.com/randovania/randovania/assets/46821362/37b498de-620c-45f6-8c5f-cf8742b41cdc)
![image](https://github.com/randovania/randovania/assets/46821362/a4322e07-6f12-45f9-9fbe-0b0f67175b82)